back/app/models/phase.rb

```diff
@@ -390,8 +388,9 @@ def set_presentation_mode
     self.presentation_mode ||= 'card'
   end
 
-  def validate_voting
-    Factory.instance.voting_method_for(self).validate_phase
+  # Delegate any rules specific to a method to the participation method itself
```

Comment:
Made this more generic so any participation method can now implement it's own validation rules